src/common/components/repository-row/index.js

```diff
@@ -1,46 +1,46 @@
 import React, { PropTypes } from 'react';
 
-import Button from '../vanilla/button';
-
-import styles from './repositoryRow.css';
+import { TR, TD } from '../vanilla/table';
+import { parseGitHubRepoUrl } from '../../helpers/github-url';
 
 const RepositoryRow = (props) => {
 
   const {
-    errorMsg,
-    repository,
-    buttonLabel,
-    buttonDisabled,
-    onButtonClick
+    snap
   } = props;
 
+  const { fullName } = parseGitHubRepoUrl(snap.git_repository_url);
+
+  return (
+    <TR>
+      <TD><a href={ `/${fullName}/builds` }>{ fullName }</a></TD>
+      <TD></TD>
+      <TD>{ renderSnapNameLabel(snap) }</TD>
+      <TD></TD>
```

Comment:
This column is for the "Latest build" status. I require clarification of how to obtain this information for one or more repositories at once in order to populate this status for all snaps in this table. (cc: @cjwatson)